src/Microsoft.Azure.WebJobs.Host/Bindings/BindingContext.cs

```diff
@@ -83,6 +86,18 @@ public TraceWriter Trace
         }
 
         /// <summary>
+        /// Gets the set of property values for the current bind operation.
+        /// </summary>
+        /// <remarks>
+        /// Bindings can add arbitrary values to this collection as needed to
+        /// flow state around.
+        /// </remarks>
+        public IDictionary<string, object> Properties
```

Comment:
What's the difference between BindingData and Properties?

---
In reply to: [95933589](https://github.com/Azure/azure-webjobs-sdk/pull/971#discussion_r95933589) [](ancestors = 95933589)